src/System.Private.ServiceModel/tools/test/TestRootCertificateInstaller/TestRootCertificateInstaller.csproj

```diff
@@ -13,8 +13,8 @@
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
```

Comment:
I could be wrong, but I think the goal was to move this project out of the test folder and put it directly under tools.  I think everything under tools\test is bound for deprecation.